testutil/integration/evmos/utils/validators.go

```diff
@@ -0,0 +1,14 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package utils
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func ConvertValAddressToHex(valAddress string) common.Address {
```

Comment:
this is for the consensus address not for the operator address
```suggestion
func ValidatorConsAddressToHex(valAddress string) common.Address {
```